client/state/selectors/is-difm-lite-website-content-submitted.ts

```diff
@@ -0,0 +1,23 @@
+import getRawSite from 'calypso/state/selectors/get-raw-site';
+import { SiteData } from '../ui/selectors/site-data';
+import type { AppState, SiteId } from 'calypso/types';
+
+/**
+ * Returns if the DIFM Lite Website Content form has been submitted.
+ *
+ * @param  {object}   state  Global state tree
+ * @param  {number}   siteId Site ID
+ * @returns {boolean}        true if submitted, false otherwise
+ */
+export default function isDIFMLiteWebsiteContentSubmitted(
```

Comment:
This function is not used in this change, but will be used for completing the task "Block access to the form if the purchase is not valid or if the form has already been submitted" (see #59895)